lib/app.rb

```diff
@@ -58,12 +60,7 @@ def run
           if !@projects || @projects.size == 0
             @output_stream.puts("\e[40;38;5;214mCan't edit any projects\e[0m")
             @output_stream.puts("\e[40;38;5;214mNo projects created\e[0m\n\n")
-            break
-          end
-
-          @output_stream.puts("\e[0;35mEnter a project name:\e[0m")
-          name = @input_stream.gets.chomp
-          if (@current_project = @projects.detect { |p| p.keys.first == name })
+          elsif switch_current_project
```

Comment:
The name of this is kind of worrisome to me. It sounds like a command, but we are also querying. It would be good to separate those two things.

Reply:
One interesting thing that is falling out of this disambiguation of query command is that it seems like we are asking for input at all different times of the loop. That isn't something we can fix right away, but interesting to note.